api/util/errorLog-function.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const router = express.Router();
+const ErrorLog = require('../models/ErrorLog');
+
+async function infoErrorLog(req){
```

Comment:
can you call this function something descriptive

```suggestion
async function addErrorLog(req){
```